lib/bindConnectors/getConfigs.js

```diff
@@ -0,0 +1,29 @@
+const AWS = require('aws-sdk');
+
+const getCurrentRegion = () => {
+	return process.env.AWS_REGION === 'dev'
+		? process.env.dev_region
+		: process.env.AWS_REGION; 
+};
+
+
+const getDatadogSecrets = async () => {
+	const ssm = new AWS.SSM({ region: getCurrentRegion() });
+	const ssmParameter = await ssm
+	.getParameter({
+		Name: process.env.DATADOG_API_SECRET_TOKEN,
```

Comment:
shouldn't there be a check somewhere that first checks `DATADOG_API_SECRET_TOKEN` exists?

Reply:
We've added a check and appropriate log if the value is missing